src/GNDStk/Node/src/child.hpp

```diff
@@ -2,27 +2,27 @@
 // -----------------------------------------------------------------------------
 // Node.child(child_t<*>)
 //
-// FIND == find::one
```

Comment:
I think `allow` is a much better word than `find`.

Reply:
Agreed. Also, "find" suggested that someone was finding content - i.e. pulling data from a GNDS tree. So, the terminology was biased towards input. "Allow", plus the name change from "all" to "many", now means: Does GNDS allow one such node under a given parent node, or many such nodes? Much more natural terminology, and I/O-agnostic.